### DISCUSSION TITLE
How to use Orchid DateTime component in blade?

### DISCUSSION THREAD

**Discussion by Muratcol** [orchidsoftware/platform#1819] [Q&A] [answered]
[Datetime](https://orchid.software/en/docs/field/#datetime) has really cool UI and I'm looking for a form field something like this.

 I wonder is it possible to use [datetime](https://orchid.software/en/docs/field/#datetime) field, into the blade component?

**Comment by tabuna** [comment, marked as answer]
You can use the usual `Blade` syntax:

```blade
{!! \Orchid\Screen\Fields\DateTimer::make('open')->title('Opening date') !!}
```

**Reply by Muratcol** [reply]
Yess, perfect. Thanks a lot @tabuna 

But how can I get the date data from the component?

I was able to get a default `selector` component like this. But `datetimer` works differently
 ```
            $('#yearSelector').on('change', function() {
                const year = $(this).val(); // get selected year
                const accountId = $('#accountSelector').val(); // Get account ID from default selector
                if (year) { // require a URL
                    window.location= `/admin/statistics/publishEngineStatistics/chart/${year}/${accountId}`;
                }
                return false;
            });
```

And this is how I call the `datetimer`:

```
                <div class="second-content">
                    <p>Year: </p>
                    
                    <div name="year" class="selectors" style="width: 150px;" id="yearSelector">
                        {!! \Orchid\Screen\Fields\DateTimer::make('yearSelector')->format('Y') !!}
                    </div>
                </div>
```

**Reply by Muratcol** [reply]
@tabuna  Do you have some advice, sir?

**Reply by tabuna** [reply]
There are a few things you should pay attention to in the above code:

```html
<div name="year" class="selectors" style="width: 150px;" id="yearSelector">
```

The div has a unique id under the name `yearSelector`. Then you are trying to hang a value change event on that div.

1. But div doesn't matter
2. There are no triggers for this to happen

You should hang the event on the field itself, not on the div.

```js
document.querySelector("[name='yearSelector']").addEventListener('change', () => {
   // code
});
```

**Reply by Muratcol** [reply]
Oh, thank you so much! I didn't know that I could reach the component with `"[name='yearSelector']"`
